6-Typescript/server/tests/it/get-all-todos.test.ts

```diff
@@ -1,29 +1,31 @@
-import { times } from 'lodash';
-import { ITodo } from '../../interfaces/ITodo';
-import { createTodo, createUserId } from '../env/random';
-import { TestKit } from '../env/TestKit';
+import { times } from "lodash";
+import { ITodo } from "../../interfaces/interfaces";
+import { createTodo, createUserId } from "../env/mockData";
+import { TestKit } from "../env/TestKit";
 describe("GET all todos /api/todos/ ", () => {
-    const testKit = new TestKit()
+    const testKit = new TestKit();
     testKit.beforeAndAfter();
 
     describe("given no todos in database", () => {
```

Comment:
you have a lot of `describe`s with 1 `it` inside them.. it's not a good practice IMO